plugins/org.jkiss.dbeaver.ui.editors.sql/src/org/jkiss/dbeaver/ui/editors/sql/SQLEditor.java

```diff
@@ -263,7 +263,7 @@ public SQLEditor()
     }
     
     public void setConsoleViewOutputEnabled(boolean value) {
-        isResultSetAutoFocusEnabled = value;
+        isResultSetAutoFocusEnabled = !value;
```

Comment:
The logic is a little bit strange. Setters and getters should be simple. But the reversed result of the setter looks strange.

Reply:
It's not a setter.